package.json

```diff
@@ -105,6 +105,7 @@
     "react-native-vosk@0.1.12": "patch:react-native-vosk@npm%3A0.1.12#./.yarn/patches/react-native-vosk-npm-0.1.12-76b1caaae8.patch",
     "eslint": "patch:eslint@8.49.0#./.yarn/patches/eslint-npm-8.39.0-d92bace04d.patch",
     "app-builder-lib@24.4.0": "patch:app-builder-lib@npm%3A24.4.0#./.yarn/patches/app-builder-lib-npm-24.4.0-05322ff057.patch",
-    "react-native@0.71.10": "patch:react-native@npm%3A0.71.10#./.yarn/patches/react-native-animation-fix/react-native-npm-0.71.10-f9c32562d8.patch"
+    "react-native@0.71.10": "patch:react-native@npm%3A0.71.10#./.yarn/patches/react-native-animation-fix/react-native-npm-0.71.10-f9c32562d8.patch",
+    "nanoid@3.3.7": "patch:nanoid@npm%3A3.3.7#./.yarn/patches/nanoid-npm-3.3.7-98824ba130.patch"
```

Comment:
In my comments I suggested keeping `nanoid@*` otherwise this will break when we update nanoid. Or maybe `nanoid@3.*` or just `nanoid`, whatever works. Currently CI is failing anyway because you need to run `yarn install` after changing the patch

Reply:
I think we need to follow semver patterns, I found it that using only `nanoid` will apply the patch to this and future releases.

Reply:
It's ok if it applies to future versions too, because those also appear to be broken. And in any case we want the patch to apply to all 3.x without having to change this line every time. So for now I think just `nanoid` is fine

Reply:
What I meant is that the patch probably needs to be applied for the `nanoid@3.3.7` because it is the one version we have specified in the dependencies of lib. If we had `nanoid@^3.3.7` would need to be applied to this version upwards, make sense? If I try to patch `nanoid@^3.3.7` instead of `nanoid@3.3.7` the patch is never applied to the lib version.

Reply:
And the problem of using only `nanoid` is that the server tests failed, I'm going to take a second look at this to be sure.